README.md

```diff
@@ -42,14 +42,25 @@
 </a>
 
 DataFusion is an extensible query engine written in [Rust] that
-uses [Apache Arrow] as its in-memory format. DataFusion's target users are
+uses [Apache Arrow] as its in-memory format.
+
+The DataFusion libraries in this repository are used to build data-centric system software. DataFusion also provides the 
```

Comment:
What about something like:

"The DataFusion ecosystem provides a suite of libraries and binaries to meet the needs of data system developers and end users (data engineers / analysts / etc).  There are libraries in this repository for those looking for a custom experience as well as the below prepackaged projects which work out of the box and target end users".

I think it puts the prepackaged projects on the same level as the libraries (not libraries and then some prepackaged projects).

Reply:
I think this sounds great. Thank you @matthewmturner 

In order to keep things moving, I would like to merge this PR as is (as it has been approved) -- would you be willing to make a follow on PR to propose adjusting the wording to your suggestion?

Reply:
Sure will do

Reply:
New PR is here: https://github.com/apache/datafusion/pull/12778